src/System.Net.NetworkInformation/src/System/Net/NetworkInformation/OsxIPGlobalProperties.cs

```diff
@@ -26,7 +26,7 @@ public unsafe override TcpConnectionInformation[] GetActiveTcpConnections()
                 TcpState state = nativeInfo.State;
 
                 byte[] localBytes = new byte[nativeInfo.LocalEndPoint.NumAddressBytes];
-                fixed (byte* localBytesPtr = localBytes)
+                fixed (byte* localBytesPtr = &localBytes[0])
```

Comment:
It looked like the native code was [always setting either 4 or 16](https://github.com/dotnet/corefx/blob/68be7c6b56008cf5c57ec4c05f96ca23526e816d/src/Native/Unix/System.Native/pal_networkstatistics.cpp#L306-L319), but to be safe, I reverted.